docs/README.md

```diff
@@ -1,401 +1,302 @@
 
 ```
-export interface AggregateAutocomplete {
-    type: string;
-    value: string;
-    score: number;
-    topic_source_id: string;
-    topic_type: string;
-    credential_id: string;
+interface AggregateAutocomplete {
+  type: string;
+  value: string;
+  score: number;
+  topic_source_id: string;
+  topic_type: string;
+  credential_id: string;
 }
 ```
 
-Results are mostly returned in batches of up to 10 in descending search `score` order. More specific query strings will result in fewer results returned with higher search `score`s. Non-matching query strings will return empty results.
+Results are mostly returned in batches of up to 10 in descending search `score` order. More specific query strings will result in fewer results returned with higher search scores. Non-matching query strings will return empty results.
 The main fields you will be interested are:
 * `type`: The data field that autocomplete search is based on (example: 'name').
 * `value`: The value of the data field that autocomplete search is based on (example: 'ABC LTD.').
 
-___
-💡 Each API resource in the Angular demo is accessed via an injectable service (for example, the `/search` resource is accessed via the [`SearchService`](/demo/angular/orgbook-autocomplete/src/app/search/services/search.service.ts).
-
-In the demo, the `getAggregateAutocomplete` method performs a `GET` request to the `/search/autocomplete` endpoint. The service method in the demo is implemented as follows:
-
-```
-public getAggregateAutocomplete(q: string): Observable<AggregateAutocompleteResponse> {
-  const queryParams = new HttpParams({
-    fromObject: { q, inactive: 'false', revoked: 'false' }
-  });
-
-  const options = { params: queryParams };
-
-  return this.http.get<AggregateAutocompleteResponse>('/search/autocomplete', options);
-}
-```
-
-Angular provides an [`HttpClient`](https://angular.io/api/common/http/HttpClient) that uses [RxJS Observables](https://rxjs-dev.firebaseapp.com/) under the hood for making HTTP requests and ties in with the framework's change detection algorithm. A similar method can be created using any HTTP service such as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), [`axios`](https://github.com/axios/axios), or even [`XMLHTTPRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest).
-
-The `getAggregateAutocomplete` method is ultimately called as a user types text into an `<input>` element in the [SearchInputComponent](/demo/angular/orgbook-autocomplete/src/app/search/components/search-input/search-input.component.ts):
+#### Example
 
-```
-<input type="text" #search name="search" id="search"
-  ...
-  (input)="onAutocomplete(search?.value)"
-  ...
-  matInput [matAutocomplete]="auto" ...>
-```
+> Checkout this [example](https://stackblitz.com/edit/js-uum64f?file=index.js) on Stackblitz for a simple implementation using jQuery and plain HTML.
```

Comment:
Great idea providing an example! It appears to not be working in Firefox (no sutocomplete results are displayed), but it works fine in Chrome as far as I can tell.